docs/content/preview/api/ycql/ddl_create_index.md

```diff
@@ -145,6 +146,39 @@ When using an index without transactions enabled, it is the responsibility of th
 
 - A unique index disallows duplicate values from being inserted into the indexed columns. It can be used to ensure uniqueness of index column values.
 
+### DEFERRED INDEX
+
+Currently, an "index backfill" job is launched for each index that is created. For the case where you create a table and add multiple indexes, the main table needs to be scanned multiple times to populate each index. This is unnecessary, and can also causes issues with the single touch and multi touch block cache algorithm.
+
+After creating a set of indexes with their backfill deferred, you can then trigger a backfill job for the entire batch of indexes (on the same table) by either:
+
+1. Creating a new index, that is not deferred:
+    ```cassandraql
```

Comment:
cassandraql doesn't appear to be recognized; use cql

Reply:
Added in https://github.com/yugabyte/yugabyte-db/pull/20523/commits/27c84ea71753d490304a76640e9aba1e268afbca